_includes/glossary/README.md

```diff
@@ -0,0 +1,38 @@
+* (Required) short description. Make sure to replace the instructional text in the template with your content.
+* (Optional) long description. If you do not provide a long description, remove the `{% capture %}` and `{% endcapture %}` tags together with the instructional text.
+
+The template also contains basic information about how to write your snippet. For additional guidance, read on in this readme.
+
+## Glossary snippet style guide
+
+This style guide supplements the guidance provided in the glossary template. It's intended to help you think about what and how to write glossary definitions. For more general guidance on style, consult [the core docs style guide](https://kubernetes.io/docs/home/contribute/style-guide/), also available in this repo at `docs/home/contribute/style-guide.md`.
+
+### Minimum viable snippet:
+
+Every snippet must include at least the short description. The long description is optional, but should be provided for terms that need additional clarification.
+
+<short-description>Required. One line (or two short lines) that provides a minimum definition. Do not repeat the term. Prefer fragments. Model after tooltips. End with a period.
+
+<long-description>Optional. Longer additional text to appear after (in conjunction with) short description. Provide in cases where the short description is not sufficient for the intro paragraph to a topic. Write complete but concise sentences.
+
+### Examples
+
+<term>Pod
+<short-description>The smallest and simplest Kubernetes object. Represents a set of running processes on your cluster.
+<long-description>A Pod most often runs only a single container, and is managed by a Deployment.
+
+<term>Deployment
+<short-description>Controller that provides declarative updates for Pods and ReplicaSets.
+<long-description>A Deployment is responsible for creating and updating instances of an application.
+
+### Thinking about definitions
+
+* Think of the short description as it would appear in a tooltip. Is it sufficient to get the reader started? Is it short enough to be read inside a small UI element? Tip: look at the `kubectl` command-line help, or at the API reference doc content -- both these deliverables pull their doc content from the generated Swagger API definition. Note, however, that this content should be used with care. The concept docs for Pod, for example, are clearer than the reference docs.
+
+* Think of the long description as following the short description in the introductory paragraph for a doc topic. Does it provide information that's not already clear from the short description? Does it provide information that readers should have a general sense of before they dive into the details of the topic it helps introduce? Tip: the long description does not need to be long; it's intended to extend but not replace the short description. Look through current related docs for ideas. (The Deployment long description is taken from a tutorial, for example.)
```

Comment:
First sentence phrasing feels a little awkward; would be awesome if the phrasing could be made a bit simpler